src/lib.rs

```diff
@@ -2148,6 +2148,27 @@ pub fn clang_version() -> ClangVersion {
     }
 }
 
+/// A ParseCallbacks implementation that will act on file includes by echoing a rerun-if-changed
+/// line
+///
+/// When running in side a `build.rs` script, this can be used to make cargo invalidate the
+/// generated bindings whenever any of the files included from the header change:
+/// ```
+/// use bindgen::builder;
+/// let bindings = builder()
+///     .header("path/to/input/header")
+///     .parse_callbacks(Box::new(bindgen::CargoCallbacks()))
+///     .generate();
+/// ```
+#[derive(Debug)]
+pub struct CargoCallbacks();
```

Comment:
Is there a good reason to have the parens? Can't this just be `pub struct CargoCallbacks;`?

Reply:
Reason? Yes, me still not having warmed to parenthesis-free unit types.
Good? No.
(I even wonder why clippy lets me off the hook here).